lib/kitchen/verifier/shell.rb

```diff
@@ -36,16 +36,23 @@ class Shell < Kitchen::Verifier::Base
       default_config :command, "true"
       default_config :shellout_opts, {}
       default_config :live_stream, $stdout
+      default_config :remote, false
```

Comment:
There are many interpretation of `remote` in test-kitchen. For example, virtualbox VM, Docker container, ec2 instance, and etc.
I think the one we should classify is only whether it's the inside or outside of the instance.